Iceberg-Plugin.package/IceGitHubPullRequestModel.class/instance/obtainAvailableBranchNames.st

```diff
@@ -0,0 +1,8 @@
+private
+obtainAvailableBranchNames
+	^ ((self repository allBranches 
```

Comment:
not really at that stage (also is not that what github does). What github does is to allow you to put a branch name if you are not convinced with the ones that he offers you. 
We can add that later, too